src/material-experimental/mdc-list/BUILD.bazel

```diff
@@ -92,6 +93,7 @@ ng_web_test_suite(
     name = "unit_tests",
     static_files = [
         "@npm//:node_modules/@material/list/dist/mdc.list.js",
+        "@npm//:node_modules/@material/ripple/dist/mdc.ripple.js",
```

Comment:
@devversion do you know why this file might be 404ing for some of the tests? It works in the dev app and all I had to do make it work for other components was to include it in the `static_files` like I'm doing here.

Reply:
@crisbeto You need to also add this to the `testing` unit tests based on what I can tell.

Reply:
Ah you're right, I didn't notice that it was complaining about the `testing` target, not the component one.